src/planner/parabolic/ParabolicTimer.cpp

```diff
@@ -41,7 +41,7 @@ std::unique_ptr<aikido::trajectory::Spline> convertToSpline(
         "computeParabolicTiming only supports Rn, "
         "SO2, and CartesianProducts consisting of those types.");
 
-  const auto interpolator = _inputTrajectory.getInterpolator();
+  auto interpolator = _inputTrajectory.getInterpolator();
```

Comment:
I think the following type check is redundant because of the one above.

Reply:
Good catch :+1: